### ISSUE TITLE
R4.2 installer autopartitioning bails partition too big

### ISSUE THREAD

**Issue by Eric678** [QubesOS/qubes-issues#8350]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### Qubes OS release
R4.2.0-rc1

### Brief summary
Selecting first SATA drive (/dev/sda) to autopartition when it is a pretty much standard 3.5" drive - 12TB, results in a failure message.

### Steps to reproduce
As above

### Expected behavior
Issue a warning message that not all the available space was allocated and use whatever it can manage on an LVM partition.

### Actual behavior
Error message and one has to resort to manual partitioning.  I did set up what I thought would work, 600M EFI FAT32, 1G /boot and 1T root encrypted LVM (also a swap that I gather was not used) and installation did get started OK.

Odd thing is that /dev/sdb worked OK, noting that the 12T root partition was set to ext4, not LVM.

**Comment by ben-grande** [comment]
Please retest on R4.3 and share the error message. I don't think Qubes has any testing machine using such a big drive to test continuously if this works. @marmarek, if the previous statement is correct, maybe indicate to @alimirjamali to announce need for bigger drives for openqa? Ideally, would allow testing all filesystems, depending on the amount of drives or reflashing continuously.

> Odd thing is that /dev/sdb worked OK, noting that the 12T root partition was set to ext4, not LVM.

What "not LVM" means? BTRFS?

**Comment by marmarek** [comment]
> I don't think Qubes has any testing machine using such a big drive to test

I don't think it needs to be a physical machine. It's trivial to have a virual one with 12TB drive (as long as the test doesn't actually try to fill all of that space). It appears to work just fine: https://openqa.qubes-os.org/tests/166488#step/install_partitioning_default/2

Maybe there was some custom option selected, like forcing MBR partition table (such large disks require GPT) ?

**Comment by Eric678** [comment]
Sorry not able to test, 4.3 still looks too fragile, and those drives are all full now.  At the time I was doing lots of default installations trying to isolate the reason for hardware resets at end of installation (was the USB controller problem).  I do remember having the impression that the problem was related to LUKS not liking that partition size.  A slightly more informative error message would have been helpful.  Perhaps now just close this.   Thanks E.

**Comment by marmarek** [comment]
FWIW, the (successful) test I linked above was on R4.2.
